packages/flutter/lib/src/material/data_table_theme.dart

```diff
@@ -49,7 +55,11 @@ class DataTableThemeData with Diagnosticable {
     this.columnSpacing,
     this.dividerThickness,
     this.checkboxHorizontalMargin,
-  });
+  }) : assert(dataRowMinHeight == null || dataRowMaxHeight == null || dataRowMaxHeight >= dataRowMinHeight),
+       assert(dataRowHeight == null || (dataRowMinHeight == null && dataRowMaxHeight == null),
```

Comment:
This assertion is causing the Google tests to fail. I need to investigate a bit.